src/NServiceBus.AcceptanceTests/Routing/When_overriding_local_address.cs

```diff
@@ -0,0 +1,76 @@
+﻿namespace ServiceBus.AcceptanceTests.Routing
+{
+    using System.Threading.Tasks;
+    using NServiceBus;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+    using Conventions = NServiceBus.AcceptanceTesting.Customization.Conventions;
+
+    public class When_overriding_local_address : NServiceBusAcceptanceTest
```

Comment:
It uses send options to send instead of routing so can be executed for all transports. `When_overriding_input_queue_name` is MSMQ-specific because the code-first instance API are MSMQ-specific.